app/src/main/res/xml/prefs_display_and_input.xml

```diff
@@ -31,6 +31,13 @@
 			android:title="@string/fullscreen"
 			app:iconSpaceReserved="false" />
 
+		<SwitchPreferenceCompat
```

Comment:
This seems like the correct place for this setting because it's related to "Display and input", and it's a less frequently changed setting